From 6a27993c64e0f50b34b53009650489a889917585 Mon Sep 17 00:00:00 2001
Message-ID: <6a27993c64e0f50b34b53009650489a889917585.1700222319.git.sharlatanus@gmail.com>
In-Reply-To: <cover.1700222319.git.sharlatanus@gmail.com>
References: <cover.1700222319.git.sharlatanus@gmail.com>
From: Sharlatan Hellseher <sharlatanus@gmail.com>
Date: Wed, 15 Nov 2023 17:03:32 +0000
Subject: [PATCH 22/32] gnu: python-extension-helpers: Improve style.

* gnu/packages/python-xyz.scm (python-extension-helpers): Improve
package style by applying indentation.

Change-Id: Icf337ea724244448a3aac8ecec8033ae3cd958e3
---
 gnu/packages/python-xyz.scm | 70 ++++++++++++++++++-------------------
 1 file changed, 35 insertions(+), 35 deletions(-)

diff --git a/gnu/packages/python-xyz.scm b/gnu/packages/python-xyz.scm
index 6d3e30a91e..3f9019d248 100644
--- a/gnu/packages/python-xyz.scm
+++ b/gnu/packages/python-xyz.scm
@@ -4244,46 +4244,46 @@ (define-public python-exceptiongroup
     (license license:expat)))
 
 (define-public python-extension-helpers
-(package
-  (name "python-extension-helpers")
-  (version "1.0.0")
-  (source
-    (origin
-      (method url-fetch)
-      (uri (pypi-uri "extension-helpers" version))
-      (sha256
+  (package
+    (name "python-extension-helpers")
+    (version "1.0.0")
+    (source
+     (origin
+       (method url-fetch)
+       (uri (pypi-uri "extension-helpers" version))
+       (sha256
         (base32 "1rjha07ds633fb81hn3i2yzk3v2flbi6qa091ix4mkvrgk3gl6ya"))))
-  (build-system pyproject-build-system)
-  (arguments
-   (list
-    #:phases
-    #~(modify-phases %standard-phases
-        (add-before 'check 'prepare-test-environment
-          (lambda _
-            ;; To solve pytest/conftest issue. Pytest tries to load all
-            ;; files with word 'test' in them.
-            ;;
-            ;; ImportError while loading conftest ...
-            ;; _pytest.pathlib.ImportPathMismatchError: ...
-            ;;
-            (call-with-output-file "pytest.ini"
-              (lambda (port)
-                (format port "[pytest]
+    (build-system pyproject-build-system)
+    (arguments
+     (list
+      #:phases
+      #~(modify-phases %standard-phases
+          (add-before 'check 'prepare-test-environment
+            (lambda _
+              ;; To solve pytest/conftest issue. Pytest tries to load all
+              ;; files with word 'test' in them.
+              ;;
+              ;; ImportError while loading conftest ...
+              ;; _pytest.pathlib.ImportPathMismatchError: ...
+              ;;
+              (call-with-output-file "pytest.ini"
+                (lambda (port)
+                  (format port "[pytest]
 python_files = test_*.py"))))))))
-  (native-inputs
-    (list python-coverage
-          python-pytest
-          python-pytest-astropy
-          python-pytest-cov
-          python-setuptools-scm))
-  (home-page "https://extension-helpers.readthedocs.io")
-  (synopsis "Astropy ecosystem utilities for building and installing packages")
-  (description
-    "The extension-helpers package includes convenience helpers to assist with
+    (native-inputs
+     (list python-coverage
+           python-pytest
+           python-pytest-astropy
+           python-pytest-cov
+           python-setuptools-scm))
+    (home-page "https://extension-helpers.readthedocs.io")
+    (synopsis "Astropy ecosystem utilities for building and installing packages")
+    (description
+     "The extension-helpers package includes convenience helpers to assist with
 building Python packages with compiled C/Cython extensions.  It is developed by
 the Astropy project but is intended to be general and usable by any Python
 package.")
-  (license license:bsd-3)))
+    (license license:bsd-3)))
 
 (define-public python-extras
   (package
-- 
2.41.0

